static/app/views/releases/list/thresholdsList.tsx

```diff
@@ -0,0 +1,136 @@
+import {useCallback, useEffect, useMemo, useState} from 'react';
+import styled from '@emotion/styled';
+
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import PanelTable from 'sentry/components/panels/panelTable';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Organization} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+
+import {Threshold, ThresholdQuery} from '../utils/types';
+
+import {ThresholdGroupRow} from './thresholdGroupRow';
+
+type Props = {
+  organization: Organization;
+  selectedEnvs: string[];
+  selectedProjects: number[];
+};
+
+function ThresholdsList({organization, selectedEnvs, selectedProjects}: Props) {
+  const api = useApi();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [_featureEnabledFlag, setFeatureEnabledFlag] = useState<boolean>(true);
+  const [thresholds, setThresholds] = useState<Threshold[]>([]);
+  const [errors, setErrors] = useState<string | null>();
+
+  const thresholdGroups: {[key: string]: {[key: string]: Threshold[]}} = useMemo(() => {
+    const byProj = {};
+    thresholds.forEach(threshold => {
+      const projId = threshold.project.id;
+      if (!byProj[projId]) {
+        byProj[projId] = {};
+      }
+      const env = threshold.environment.name;
+      if (!byProj[projId][env]) {
+        byProj[projId][env] = [];
+      }
+      byProj[projId][env].push(threshold);
+    });
+    return byProj;
+  }, [thresholds]);
+
+  const fetchThresholds = useCallback(async () => {
```

Comment:
what about the whole useFetchThresholdsListData thing

Reply:
Oh oof thanks - i think i copy'd from my non-pushed changes.
will update